user-guide/Advanced_Functionality/Databases/Configuring_dedicated_clustered_storage/Cassandra-compatible database service/Maintenance_Cassandra/Maintain_Cassandra_cluster.md

```diff
@@ -4,6 +4,17 @@ uid: Maintain_Cassandra_Cluster
 
 # Maintaining a Cassandra cluster
 
+## Setting the heap space
+
+To ensure that your Cassandra database does not run out of memory under load, in the *jvm-server.options* file, configure the parameters **Xmx** and **Xms** to set the Cassandra heap space.
```

Comment:
Could you add an example of this configuration here? That would make this section a bit more user-friendly.

Reply:
As this PR is already an improvement as it is now, I will already merge this. However, could you remember to add the example later?